examples/visualization/styles/helpers/size-bins-style.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>GeoJSON Layer with CARTO Tiles Example</title>
+
+    <!-- Custom CSS -->
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://api.tiles.mapbox.com/mapbox-gl-js/v1.0.0/mapbox-gl.css"
+    />
+
+    <style>
+      body {
+        margin: 0;
+        padding: 0;
+      }
+
+      #map {
+        width: 100vw;
+        height: 100vh;
+      }
+    </style>
+  </head>
+
+  <body>
+    <section id="map"></section>
+
+    <!-- Map libraries-->
+    <script src="https://api.tiles.mapbox.com/mapbox-gl-js/v1.8.1/mapbox-gl.js"></script>
+    <script src="https://unpkg.com/deck.gl@8.1.0/dist.min.js"></script>
+
+    <!-- CARTO Deck.gl's Tile Layer -->
+    <script src="/packages/toolkit/dist/umd/index.min.js"></script>
+
+    <script>
+      async function initialize() {
+        carto.setDefaultCredentials({ username: 'public' });
+
+        const deckMap = new window.deck.DeckGL({
+          container: 'map',
+          mapStyle:
+            'https://basemaps.cartocdn.com/gl/voyager-gl-style/style.json',
+          initialViewState: {
+            longitude: -3.700385991742541,
+            latitude: 40.41532659629741,
+            zoom: 14
+          },
+          controller: true
+        });
+
+        const availabilityStyle = carto.viz.sizeBinsStyle('availability_365', {
+          bins: [0, 100, 200, 300, 365],
+          sizes: [5, 20, 50, 75]
+        });
+
+        const airbnbLayer = new carto.viz.Layer(
+          'listings_madrid',
+          availabilityStyle
+        );
+        await airbnbLayer.addTo(deckMap);
+
+        const distanceStyle = carto.viz.sizeBinsStyle('distance', {
+          bins: [0, 500, 700, 900],
+          sizes: [2, 5, 10]
+        });
```

Comment:
This is the https://github.com/CartoDB/cartoframes/blob/v1.0.2/cartoframes/viz/styles/size_bins_style.py equivalent in CF, in case it is useful